.changelog/unreleased/improvements/1049-deprecate-mockcontext-withclient-methods.md

```diff
@@ -0,0 +1,3 @@
+- [ibc-testkit] Deprecate `MockContext::with_client*` in favor of
```

Comment:
A nit here. The name of it `.md` should get fixed.

Reply:
ah. you mean `1049-deprecate-mockcontext-with-client-methods.md`, right?